Resources/Prototypes/Entities/Structures/Dispensers/booze.yml

```
@@ -45,3 +45,9 @@
     - id: DrinkVodkaBottleFull
     - id: DrinkWhiskeyBottleFull
     - id: DrinkWineBottleFull
+# Moffstation Start - Dispenser QOL changes
+    - id: DrinkMelonLiquorBottleFull
+    - id: DrinkBlueCuracaoBottleFull
+    - id: DrinkAbsintheBottleFull
+    - id: DrinkChampagneBottleFull
+# Moffstation End
```

```suggestion
    # Moffstation - Begin - Dispenser QOL changes
    - id: DrinkMelonLiquorBottleFull
    - id: DrinkBlueCuracaoBottleFull
    - id: DrinkAbsintheBottleFull
    - id: DrinkChampagneBottleFull
    # Moffstation - End
```
Moffstation block guards should be indented to the same level as the content they're guarding.